roles/openshift_adm/tasks/api_cert.yml

```diff
@@ -44,13 +44,6 @@
   loop_control:
     label: "{{ item }}"
 
-- name: Wait until the OpenShift cluster is stable after certificate update.
-  environment:
-    KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
-    PATH: "{{ cifmw_path }}"
-  ansible.builtin.command:
-    cmd: "oc adm wait-for-stable-cluster --minimum-stable-period=3m"
-
```

Comment:
I'm a bit unsure about this.

This step is required by the [procedure](https://access.redhat.com/articles/regenerating_cluster_certificates) of the certificates regeneration.

The command itself doesn't simply check for the cluster operators readiness, but it ensure their stability in a defined timeframe, as you can guess by the command line argument and from the [source code](https://github.com/openshift/oc/blob/release-4.16/pkg/cli/admin/waitforstable/command.go) of the command itself.

It's true that the cluster won't be used as-is, but what happens if we introduce an issue given by the new certificates?

Reply:
that's why I'm testing it right now. Thing is, the certificate is regenerated once we boot the cluster from the overlays. I really don't think this command should be placed here.

If really, it should be done once the certificates are really re-generated. Not before shutting the cluster down. Note that we have the same command, but with a shorter minimum-stable-period:
https://github.com/openstack-k8s-operators/ci-framework/blob/6148ebc68a63451f9e7701b04a03d79649596122/roles/openshift_adm/tasks/wait_for_cluster.yml#L62-L68

And that one is called at the "right" place.

Reply:
after some discussions : we seem to need to keep it - but I'll check the doc to see the default value for the minimum-stable-period - the doc doesn't set any parameter for that one